db/column_family.h

```diff
@@ -735,6 +737,7 @@ class ColumnFamilySet {
   WriteController* write_controller_;
   BlockCacheTracer* const block_cache_tracer_;
   std::shared_ptr<IOTracer> io_tracer_;
+  std::string db_id_;
```

Comment:
I guess this should also be a reference (for the same reason as the member in `BlobSource`)

Reply:
Wow! Thanks.